packages/core/src/stepFunctionsWorkflowStudio/WorkflowStudioEditor.ts

```diff
@@ -0,0 +1,180 @@
+/*!
```

Comment:
> Also you don't need to capitalize file names

Yes. In general, please notice the conventions of the codebase so that we don't have to mention them in the PR review. That sames time. 

Please review https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#develop and the documents linked there.

Reply:
I was thinking of having it separately as this is quite different from existing SFN integrations, but I belive having this logic in a subfolder (as you suggested) should work too, as long as we keep it completely separated from existing SFN actions

Reply:
> having this logic in a subfolder (as you suggested) should work too,


sounds good.